modules/reference/pages/rpk/rpk-topic/rpk-topic-consume.adoc

```diff
@@ -10,293 +10,317 @@ The default output format `--format json` is a special format that outputs each
 record as JSON. There may be more single-word-no-escapes formats added later.
```

Comment:
@Deflaimun I suggest removing these sentences, if possible: "There may be more single-word-no-escapes formats added later. Outside of these special formats, formatting follows the rules described below."

Reply:
removed